src/test/java/org/opensearch/performanceanalyzer/reader/ReaderMetricsProcessorTests.java

```diff
@@ -53,6 +54,42 @@ public void before() throws Exception {
         rootLocation = "build/resources/test/reader/";
     }
 
+    @Test
+    public void testShardBulkParseAndQuery() throws Exception {
+        deleteAll();
+        ReaderMetricsProcessor mp =
+                new ReaderMetricsProcessor(rootLocation, true, new AppContext());
+
+        mp.processMetrics(rootLocation, 1566413985000L);
+        mp.processMetrics(rootLocation, 1566413990000L);
```

Comment:
I don;t see any `shardbulk` metric within the timestamp file below. How is this test working? 

```
^threads/21543/shardbulk/23632396/start
current_time:1674690064926
StartTime:1674690064926
ItemCount:8
IndexName:index-1
ShardID:2
```

Reply:
The `shardbulk` metrics are in the first one (which was already in resources folder). The second processMetrics invocation is for the ReaderMetricsProcessor to emit the Metrics from the previous run. On each run, emitMetrics emits Snapshots from currWindowStartTime - MetricsConfiguration.SAMPLING_INTERVAL timestamp.
https://github.com/opensearch-project/performance-analyzer-rca/blob/main/src/main/java/org/opensearch/performanceanalyzer/reader/ReaderMetricsProcessor.java#L361